packages/react-core/src/components/Chip/__tests__/ChipGroup.test.tsx

```diff
@@ -6,65 +6,63 @@ import userEvent from '@testing-library/user-event';
 import { ChipGroup } from '../index';
 import { Chip } from '../../Chip';
 
-describe('ChipGroup', () => {
-  test('chip group default', () => {
-    const { asFragment } = render(
-      <ChipGroup>
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
+test('chip group default', () => {
+  render(
+    <ChipGroup>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
 
-    expect(asFragment()).toMatchSnapshot();
-  });
+  expect(screen.getByRole('group')).toHaveClass('pf-v5-c-chip-group');
+});
 
-  test('chip group with category', () => {
-    const { asFragment } = render(
-      <ChipGroup categoryName="category">
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('chip group with category', () => {
+  render(
+    <ChipGroup categoryName="category">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByText('category')).toBeVisible();
+});
 
-  test('chip group with closable category', () => {
-    const { asFragment } = render(
-      <ChipGroup categoryName="category" isClosable>
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('chip group with closable category', () => {
+  render(
+    <ChipGroup categoryName="category" isClosable>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByLabelText('Close chip group').closest('div')).toHaveClass('pf-v5-c-chip-group__close');
+});
 
-  test('chip group expanded', async () => {
-    const user = userEvent.setup();
+test('chip group expanded', async () => {
```

Comment:
This test could be split into several separate tests:

- the overflow chip does not render by default when less than 3 children are passed
- the overflow chip renders with the default `numChips` prop value of 3 (this could probably be the baseline of the overflow chip being collapsed by default as well)
- the overflow chip renders when `numChips` is passed in a value other than 3 (passing in 1 or 2 might be better?)
- clicking the overflow chip causes the text to update with the default props
- passing in the `expandedText` and `collapsedText` props work
- passing `defaultIsOpen` of true causes the chip group to be expanded by default